Description: Remove call to git log when building sphinx docs
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-03-02

--- python-oslo.messaging-4.5.0.orig/doc/source/conf.py
+++ python-oslo.messaging-4.5.0/doc/source/conf.py
@@ -54,10 +54,10 @@ html_static_path = ['static']
 # Output file base name for HTML help builder.
 htmlhelp_basename = '%sdoc' % project
 
-git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
-           "-n1"]
-html_last_updated_fmt = subprocess.check_output(git_cmd,
-                                                stdin=subprocess.PIPE)
+#git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
+#           "-n1"]
+#html_last_updated_fmt = subprocess.check_output(git_cmd,
+#                                                stdin=subprocess.PIPE)
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass
